rules/falco_rules.yaml

```diff
@@ -1172,6 +1172,17 @@
 - macro: calico_node
   condition: (container.image.repository endswith calico/node and proc.name=calico-node)
 
+# Return always_true from this macro to get warnings about the cups daemons
+# regular updates in /etc/cups/.
+- macro: cupsd_state_updates
```

Comment:
I'm not sure if this macro is actually useful. In the ruleset we have already never_true macros and I would avoid adding a new one if not strictly required.